apps/ledger-live-mobile/detox.config.js

```diff
@@ -70,6 +70,8 @@ module.exports = {
       device: {
         avdName: "Pixel_5_API_31",
       },
+      gpuMode: "host",
```

Comment:
What's this doing? Maybe worth a comment to explain

Reply:
Enabled hardware acceleration, it's in the doc